internal/ingress/annotations/parser/main.go

```diff
@@ -29,20 +29,79 @@ import (
 )
 
 // DefaultAnnotationsPrefix defines the common prefix used in the nginx ingress controller
-const DefaultAnnotationsPrefix = "nginx.ingress.kubernetes.io"
+const (
+	DefaultAnnotationsPrefix          = "nginx.ingress.kubernetes.io"
+	DefaultEnableAnnotationValidation = true
```

Comment:
the default value is `false` , right? 
```suggestion
	DefaultEnableAnnotationValidation = false
```

Reply:
nope, the global default is true, but we disable it by the flag. It is fine for now IMO

Reply:
ok